.github/workflows/release.yml

```diff
@@ -0,0 +1,23 @@
+name: Release new version
+
+on: workflow_dispatch
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-node@v4
+        with:
+          node-version-file: ".nvmrc"
+          registry-url: https://registry.npmjs.org/
+      - run: npm ci
+      - run: npm run lint
+      - run: npm run build
+      - name: Semantic Release
+        uses: cycjimmy/semantic-release-action@v4
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          NPM_TOKEN: ${{ secrets.NPM_TOKEN }}
+          GIT_AUTHOR_NAME: "ci"
+          GIT_AUTHOR_EMAIL: "deploy@monday.com"
```

Comment:
If it's open source maybe worth using a different email or something more specific to Vine?

Reply:
It is the same git user being used in the other libs, so I went with this convention